### ISSUE TITLE
When png with transparent background used for inpainting, Fooocus stuck at `image processing`

### ISSUE THREAD

**Issue by lanyusan** [MoonRide303/Fooocus-MRE#109]
```
LoRAs loaded: [('sd_xl_offset_example-lora_1.0.safetensors', 0.5), ('None', 0.5), ('None', 0.5), ('None', 0.5), ('None', 0.5)]
Fooocus Expansion engine loaded for cuda:0.
App started successful. Use the app with http://127.0.0.1:7860/ or 127.0.0.1:7860
[Fooocus] Image processing ...
```

Image used for testing:

![medium](https://github.com/MoonRide303/Fooocus-MRE/assets/56706512/46ac8ea6-61ec-4688-ab3a-f684ff727a3c)

**Comment by lanyusan** [comment]
It was meant to be posted to Fooocus. I will leave it here.

**Comment by MoonRide303** [comment]
Yeah, it's was the same problem as [#462 in vanilla](https://github.com/lllyasviel/Fooocus/issues/462). Getting stuck fixed in vanilla v2.0.77 (merged to MRE), but approach to alpha channel might need re-thinking.

**Comment by lanyusan** [comment]
@MoonRide303 

Yes it works now. Just as you said, the transparent background was turned to pure black so when use inpaint to repaint whole background, the repainted background was very dark too.

Maybe transparent background should be randomized? 

![image](https://github.com/MoonRide303/Fooocus-MRE/assets/56706512/51f24e22-9a4b-4c19-93aa-c1801f969ac4)
